<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FReportByPayment</class>
 <widget class="QWidget" name="FReportByPayment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date from</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="leDateFrom"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Date to</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EDateEdit" name="leDateTo"/>
   </item>
   <item>
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>Branch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leBranch">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="ShowButton" stdset="0">
      <bool>true</bool>
     </property>
     <property name="EnableHiddenText" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hall</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="EQLineEdit" name="leHall">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="ShowButton" stdset="0">
      <bool>true</bool>
     </property>
     <property name="EnableHiddenText" stdset="0">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>170</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
